notebooks/tutorials/mass_wasting_runout/landslide_runout_animation.ipynb

```diff
@@ -0,0 +1,567 @@
qokSv1hilinijngkjmhmjSWGaJ1kilikUPHJHIhKSRyIQyOpKspDKSCDXmXxg+Cfwk/aA8Eal8N/jT8PPCvxL8EaqM3Xh/wAV6VBqVrHcKjxw6hp0zqt5o+r2gkdrDWtIubHVtPkbzrK9t5QHH5PS/sb/ALaf7Bksmu/8E8/ik/xx+BVk73N5+xD+0f4jnvU0mwDF5NO+BHxfvZY73wrIivImk+H/ABHdWehwv5moa5feLNQeO3Jqt9V3W/3dfl9xVoy2919n8L9HuvR3XmftlRX5zfsxf8FNPgP+0B4rf4NeNrHxP+zP+09pkkdlr37Ofx4sj4Q8aPqTDAXwTqWoR2ekfEDT7srLPpEmivFrl/piDVZvDljZSxyN+jNNNPYlpp2at/XTo/k2FfkB/wAEROf2L9bb+/8AtK/tGt9c/EbUBx+Vfr/X5Af8EQef2Jr5x0k/aL/aKcH1B+JWqDP6d+aT+Jej/Qa+CXrD/wBuP1/ooopkhRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABX4/2PH/AAXq1fHf/glDCx+o/ax0wD9K/YCvyBteP+C9Oo/7f/BJ1P0/a0sP/iaT6eq/JlR+1/gl/wC2n9A1FIOQD6gUtYGwUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQAUUUUAFFFFABRRRQB/nL/APBfbV/2g/i7/wAFLvjdomr6d8QfF3gT4UReC/Bfwq0+y8NatdeG/DXhu88AeE/E+s22i/2dpxtJ7jVPFetazqOsapNJdald3LxWNxdmw0nSrGw+Bf2X/wBoD9u/9jDUvF+s/syzfEX4Wax48sdJ0zxVqtj8ItH8Q6lqWm6JcX11p+nx33jDwN4hutMsRdahPc3lppMtjb6pcRafNqkd5JpWltZ/6sVFPTz/AALU9LW/H/gH+bB/w99/4LYf9F2+M/8A4Yz4d/8Azp6/rG/4ID/tM/tWftRfsy/GDxj+1x4w8U+MvHOgfHa98M+Hb/xZ4R0TwdfWvhOP4f8AgbVYbS30/QvDnhm2ubQaxqWrTC8ls552mllhNyY4Uii/dmilp5ick1ZRS8/6QUUUUEhRRRQB/IF/wRRPmf8ABRL/AIK4S/3/AIw+IWz67/jz8ajk/l/9av6da/mG/wCCIZ8z9v8A/wCCs03B3/FrVGz/ANdPjn8bm/L5f5V/TzW0PhXzM6vx/KP/AKSgoooqjMKKKKACiiigAooooAKKKKACiiigAooooAK/ID/glTz8Q/8AgqW3r/wUu+PK5/3bTw4Mfhn9a/X+vx//AOCUvPjz/gqS3r/wU2/aDXP+7b+Ghj8M/wCeaT3j6v8AIqPwz9I/+lH7AV80/tMfsgfs6ftfeEB4N+P/AMMdB8b2tokx0DX3jk0vxp4RupRu+3+EfGOmPa+INAuBMsU80NlfLp+oPDFFqtlf2oa3b6WopkptO6dn3R+JP/CB/wDBSH/gnjib4Ta5q/8AwUa/ZS0r73wt8f6lDp37W3w10GAbUg8FeNlgey+LFnptog8nR7+yl1m6Ednonhvw3o9sJdRH3N+yl+35+zT+2HbXth8LvGM+kfEjQBNF40+CPxEsG8FfGTwRfWb+VqNnrvgnU5TdXUemz4t77VvD0+t6FbXLC0m1RLwSW6faFfDP7Vv/AATx/Zt/a4uLPxX4x8P6p4C+NGgGC48GftCfCTU38BfGfwlqNkmzTLu18XaVGJNah07G2z0/xLb6vZ2KPK+lpp1263aKzW2vk/0f+dy7xl8Ss/5or847P1Vn6n3NRX4lD4z/APBRH/gnr/of7SnhDUf2+P2XNL+SP9oj4NaFDp/7Rnw/0OE7Vu/ir8KRcfZPGVnp9nEZL3xBoV4XgtYrnWvE3i65vZo9NP6afs6/tT/s/wD7WPgmL4gfs/8AxP8ADXxG0DbAupQ6Vcvb+IPDd3cIzx6b4s8L6hHaeIfC+pMqO0dnrem2UlxEv2i0+0WrxzuJp6bPs/61+QnFrXdd1qvn1T8mkfQNFFFMkKKKKACiiigAooooAKKKKACiiigAooooAKKKKACiiigAooooAKKKKAKOqaXpmt6bf6PrWnWOr6RqlpcWGp6Vqlpb3+m6jYXcTQXVlf2N3HLa3dpcwu8NxbXEUkM0TtHIjIxB/Inx/wD8Eu9X+EXi3WfjT/wTR+MF5+yL8S9TuTqniP4PXUVz4j/ZV+KN2gA+yeJvhtKl7H4LnuIs2kGu+ErK5j0GzLp4d8PaZdzPqCfsJRSaT/rX7xqTWz0e6eqfqno/61PyB+HP/BUe6+GPjHSPgj/wUj+EV9+xx8WNSm/s/wAPfEye4l179lz4pXCHBvvB/wAU4nvLLwq8yYu7jRfFt/LDoFs8EWueJrfUp109P1y0/ULDVrCy1XSr6z1PTNStbe+07UdPuYb2wv7K7iWe1vLK8tnlt7q1uYHSa3uIJJIZonWSN2RgTyfxG+Gfw9+L3g/V/h/8UvBXhj4g+CdegNvq/hfxfothr2i3qAHy5JLHUIJ4UurZz51neRLHd2VwqXNpPBcRpIv5G6h+wN+0/wDsV3954x/4JjfGH7X8PPtc+qaz+w9+0NrWp+J/hJqKSytPeW3wl8d3923iP4b6rcAt9lt7/U1tNQ1KWO68ReK3021j00mq81+P/B/Mr3Zbe4+zu4v0e8fndeZ+1VfNf7S/7IX7Ov7Xvg//AIQv4/8Awx0HxzZ2yTHQdcljk03xl4Su5gD9v8I+MNMe18QeH7kSrFNNFY36WOoGGOHVbO/tA9u/yz+zz/wVF+EXxJ8bw/Af9oXwp4n/AGN/2pIWgtrn4L/HExaTY+JLudzFDcfC/wCIssVl4X8e6VqMwEWiSRvpOq665ZtE0nU7SP7a/wCnVGj8/wCuxNpRfVPo/wDJrR/Js/En/hX3/BSD/gnjmf4Qa9q//BRj9lPS+T8JviJqkWnftZ/DfQosKtv4H8dLbSW
```

Comment:
Please change 
"Nontheless" as "Nonetheless"
"moderatly" as " moderately"
"uplsope" as "upslope"
"untill" as "until"

Reply:
I cleaned up the two notebooks